pkg/webui/components/icon/index.js

```diff
@@ -44,6 +44,8 @@ const hardcoded = {
   event_update: 'edit',
   event_uplink: 'arrow_drop_up',
   event_downlink: 'arrow_drop_down',
+  uplink: 'trending_up',
```

Comment:
I think we should use `arrow_drop_{up|down}` for all up-/downlink related matters.

```diff
- event_uplink: 'arrow_drop_up',
- event_downlink: 'arrow_drop_down',
+ uplink: 'arrow-drop-up',
+ downlink: 'arrow-drop-down',
```